src/app/js/formats/hierarchy/Hierarchy.js

```diff
@@ -727,59 +698,71 @@ class Hierarchy extends PureComponent {
                 }}
                 id={`divContainer${this.uniqueId}`}
             >
+                {/* 1) MouseIcon */}
```

Comment:
All this MouseIcon part could be made a React Component, so that it could be reused in other formats...

Reply:
Just added MouseIcon and CenterIcon in the `shared` folder.